packages/terra-horizontal-divider/src/HorizontalDivider.scss

```diff
@@ -0,0 +1,13 @@
+@import './variables';
+@import './mixins';
+
+:local {
+  .divider {
+    border: 0 solid;
+    border-color: var(--terra-horizontal-divider-border-color, #b2b5b6);
+    border-top: 1px solid #dedfe0;
+    border-width: var(--terra-horizontal-divider-border-width, 1px);
+    height: 0;
+    margin: 0;
+  }
+}
```

Comment:
Delete _mixins.scss and _variables.scss files if we don't use them.

Reply:
Updated https://github.com/cerner/terra-core/commit/b6a950329f831a533a9f38d54427d9c17a5e17e6#diff-23c60d19275a3c865f09b7cdc50dbcd8R10